internal/page/data.go

```diff
@@ -394,6 +394,20 @@ func (l *Lpa) TrustCorporationsNames() []string {
 	return names
 }
 
+func (l *Lpa) FeeAmount() int {
+	paid := 0
+
+	for _, payment := range l.PaymentDetails {
+		paid = paid + payment.Amount
```

Comment:
+= does exist in Go

Reply:
Haha I love that I can hear the tone in this comment 😂  noted ✅